vault/changelog.release.2022-10-25.md

```diff
@@ -0,0 +1,113 @@
+---
+id: t5o6gd1p96q1c5v17r94lfu
+title: '0.116'
+desc: ''
+updated: 1666717134811
+created: 1666620917865
+---
+
+Dendron 0.116 has sprouted  🌱
+October 25, 2022
+
+## Everything Else
+
+- feat(workspace): copy as command ([[docs|dendron://dendron.dendron-site/dendron.ref.commands]])
+- enhance(sync): selection export scope for pods v2 ([[docs|dendron://dendron.dendron-site/dendron.ref.commands]])
+- fix(workspace): go to definition for wikilink with header
+- fix(lookup): cancel note creation during "Create Note with Template" if template was not selected
+- fix(views): task notes inside of note references should render correctly
+- fix(workspace): custom color decoration for hashtags
+- fix(commands): lookup sometimes omits last keystrokes in new note when under load
+- fix(publish): use fuzzThreshold config option for search
+- fix(publish): bad breadcumbs display
+- fix(publish): runtime error on undefined dendron config
+
+
+## Community
+
+### General Updates
+
+**Greenhouse this Friday!**
+
+We have a [[Greenhouse Talk|dendron://dendron.dendron-site/community.events.greenhouse]] this week! Let's listen to an amazing speaker, Ryan Hill, speak on '10,000 tables when all you need is a note' 
```

Comment:
This week's [[Greenhouse Talk|dendron://dendron.dendron-site/community.events.greenhouse]] talk: {TITLE} - by {PRESENTER} (social media link)

ABSTRACT